.vsts-dotnet.yml

```diff
@@ -10,12 +10,28 @@ trigger:
 #   SignType: real
 #   SkipApplyOptimizationData: false
```

Comment:
I considered promoting the other vars but realized I only ever changed the value of `SkipApplyOptimizationData` once ever 🤷‍♂️

Reply:
It doesn't appear to have worked so looks like you were totally right. https://dev.azure.com/devdiv/DevDiv/_build/results?buildId=4728100&view=logs&j=bb592630-4b9d-53ad-3960-d954a70a95cf&t=bb592630-4b9d-53ad-3960-d954a70a95cf

🙇🏻‍♂️ 

That one is helpful to bootstrap from nothing to get a build from which you can collect updated optprof data. But in no other circumstance. And SignType is basically never helpful. So I'm going to stop bothering.

Reply:
For whoever decides to do this in the future:

I think it's because the call to CIBuild.cmd needs to be updated to take the pipeline parameter. Or some script variable needs be created to take in that value?

Reply:
the call already takes the pipeline parameter, right? My untested pet theory was that it was because it was defined twice, once in the YAML and once in the AzDO metadata.

Reply:
I want to say that's a runtime variable and that syntax doesn't work with compile time variables and blarg 🙃

Reply:
Bingo bango! #6415